src/lib/dates.js

```diff
@@ -49,6 +49,85 @@ function isWorldCalendar(calendar) {
 // of the unix epoch. From calendars.instance().newDate(1970, 1, 1).toJD()
 var EPOCHJD = 2440587.5;
 
+// each calendar needs its own default canonical tick. I would love to use
+// 2000-01-01 (or even 0000-01-01) for them all but they don't necessarily
+// all support either of those dates. Instead I'll use the most significant
+// number they *do* support, biased toward the present day.
+var CANONICAL_TICK = {
```

Comment:
maybe the world-calendars *plotly.js component*... the package is a pretty thin wrapper on kbwood/calendars so in principle I'd like it to stay in sync with the original.